src/Docker/Runner/Environment.php

```diff
@@ -7,58 +7,30 @@
 
 class Environment
 {
-    /**
-     * @var string
-     */
-    private $configId;
+    private ?string $configId;
+    private array $configParameters;
+    private array $tokenInfo;
+    private ?string $runId;
+    private string $url;
+    private Component $component;
+    private string $stackId;
+    private ?string $configRowId;
+    private string $token;
+    private ?string $branchId;
+    private ?string $mlflowAbsConnectionString;
 
-    /**
-     * @var array
-     */
-    private $configParameters;
-
-    /**
-     * @var array
-     */
-    private $tokenInfo;
-
-    /**
-     * @var string
-     */
-    private $runId;
-
-    /**
-     * @var string
-     */
-    private $url;
-
-    /**
-     * @var Component
-     */
-    private $component;
-
-    /**
-     * @var string
-     */
-    private $stackId;
-
-    /**
-     * @var string
-     */
-    private $configRowId;
-
-    /**
-     * @var string
-     */
-    private $token;
-
-    /**
-     * @var string
-     */
-    private $branchId;
-
-    public function __construct($configId, $configRowId, Component $component, array $config, array $tokenInfo, $runId, $url, $token, $branchId)
-    {
+    public function __construct(
+        ?string $configId,
+        ?string $configRowId,
+        Component $component,
+        array $config,
+        array $tokenInfo,
+        ?string $runId,
+        string $url,
+        string $token,
+        ?string $branchId,
+        ?string $mlflowAbsConnectionString = null
```

Comment:
ten default bych dal pryč, to je zralý na to, že se na něj někde zapomene

Reply:
současně tě to donutí upravit testy Environment classy

Reply:
jo, dobrej napad :+1: upraveno